packages/libs/react-components/tsconfig.json

```diff
@@ -2,20 +2,7 @@
   "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
   "compilerOptions": {
     "types": ["heft-jest"],
-    "target": "ES2017",
-    "esModuleInterop": true,
-    "strict": true,
     "skipLibCheck": true,
-    "jsx": "react",
-    "module": "ESNext",
-    "declaration": true,
-    "declarationDir": "types",
-    "sourceMap": true,
-    "outDir": "dist/esm",
-    "allowJs": true,
-    "moduleResolution": "node",
-    "allowSyntheticDefaultImports": true,
-    "forceConsistentCasingInFileNames": true
-  },
-  "include": ["src/**/*"]
+    "outDir": "lib"
+  }
 }
```

Comment:
what was the reasoning for removing this? I believe this change will break the usage of this package since package.json is still pointing at `dist/esm`

Reply:
I have no idea...I was desperately trying to get the build to run